master/master.cfg

```diff
@@ -673,42 +673,42 @@ c['builders'] = []
 # breakages on various platforms if they are discovered late.
 create_builder('arm32-linux-32', LLVM_TRUNK_BRANCH)
 create_builder('arm64-linux-64', LLVM_TRUNK_BRANCH)
-create_builder('arm32-linux-32', LLVM_8_BRANCH)
-create_builder('arm64-linux-64', LLVM_8_BRANCH)
+create_builder('arm32-linux-32', LLVM_9_BRANCH)
+create_builder('arm64-linux-64', LLVM_9_BRANCH)
 
 create_builder('linux-32-gcc53', LLVM_TRUNK_BRANCH)
 create_builder('linux-64-gcc53', LLVM_TRUNK_BRANCH)
-create_builder('linux-32-gcc53', LLVM_8_BRANCH)
 create_builder('linux-32-gcc53', LLVM_9_BRANCH)
-create_builder('linux-64-gcc53', LLVM_8_BRANCH)
+create_builder('linux-32-gcc53', LLVM_10_BRANCH)
 create_builder('linux-64-gcc53', LLVM_9_BRANCH)
+create_builder('linux-64-gcc53', LLVM_10_BRANCH)
 
 create_builder('mac-64', LLVM_TRUNK_BRANCH)
-create_builder('mac-64', LLVM_8_BRANCH)
 create_builder('mac-64', LLVM_9_BRANCH)
+create_builder('mac-64', LLVM_10_BRANCH)
 
 create_builder('win-32', LLVM_TRUNK_BRANCH)
 create_builder('win-64', LLVM_TRUNK_BRANCH)
-create_builder('win-32-distro', LLVM_8_BRANCH)
-create_builder('win-64-distro', LLVM_8_BRANCH)
+create_builder('win-32-distro', LLVM_9_BRANCH)
```

Comment:
These shouldn't have been 8 - they should have been the most recently released llvm. So let's update all of these 8's to 10's

Reply:
I think I was unclear: I think the win distros and the testbranch builders should all be on the newest released llvm, so 10. We should leave a single testbranch builder on 9 (the one below I commented on) to make sure we don't break things on older llvm versions.